src/zkscript/fields/fq2_over_2_residue_equal_u.py

```diff
@@ -17,7 +18,24 @@ def square(
         clean_constant: bool | None,
         is_constant_reused: bool | None = None,
     ) -> Script:
-        """Squaring in Fq4 = F_q^2[v] / (v^2 - u)."""
+        """Perform squaring in Fq4 = F_q^2[v] / (v^2 - u).
+
```

Comment:
Add stack input, stack output, raises sections in documentation

Reply:
Done